### ISSUE TITLE
[QUESTION] How to use my own training data

### ISSUE THREAD

**Issue by manbharae** [bapalto/birdsong-keras#5]
I have two folders. Each folder has one type of sound. (about 100 1 second samples each type) 
Please suggest how to train model on my data.

**Comment by bapalto** [comment]
If the audio is not sampled at 16kHz than you have to resample the wave files with the "resample_wavs_to_16k.sh" (preprocess folder) script first.

Next you have to modify the "loadData.py" script in the preprocess folder. You may eliminate every XML related parts and create a Pandas DataFrame that contains the proper labels of the corresponding folders.
Further modifications might be necessary in "train" and "predict" folders.
